src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -282,14 +282,17 @@
                                           IsEnabled="{x:Bind Enabled, Mode=OneWay}"
                                           Margin="{StaticResource SmallTopMargin}"/>
 
-
+                                <CheckBox x:Uid="PowerLauncher_ConfirmSystemActions"
+                                        IsChecked="{x:Bind Path=ConfirmSys, Mode=TwoWay}"
+                                        IsEnabled="{x:Bind Enabled, Mode=OneWay}"
+                                        Margin="{StaticResource SmallTopMargin}"/>
```

Comment:
It shouldn't be here. It should be under System plugin

Reply:
So, let's remove it

Reply:
Done

Reply:
@mykhailopylyp - I am having some trouble here.  I'm not sure where you want this to go.  When I remove it from here, obviously it no longer shows up as an option.  I can't find a similar file under the System plugin.

If you are referring to implementing the settings interface, I have done that, but I must have missed something somewhere, because the checkbox still does not show up under the settings, and now I don't have access to CurrentPluginMetadata.ConfirmSystemCommands or CurrentPluginMetadata.ConfirmSys, and I'm not sure how to get it back.  

I have committed/pushed all the changes that I have so far, and would really appreciate your help figuring this out.

Thanks

Reply:
> I'm not sure where you want this to go

A checkbox is dynamically rendered based on `AditionalOptions`

Reply:
Does the solution build successfully for you? It failed to build on the pipeline.

Reply:
```
src\modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.System\Main.cs(213,53): Error CS1061: 'PluginMetadata' does not contain a definition for 'ConfirmSystemCommands' and no accessible extension method 'ConfirmSystemCommands' accepting a first argument of type 'PluginMetadata' could be found (are you missing a using directive or an assembly reference?)
```

Reply:
Try to understand how `Disable drive detection warning` is rendered.

![image](https://user-images.githubusercontent.com/17161067/126513401-47a81e4a-c63a-49d3-aa0b-eb5a1e9fcb4e.png)

https://github.com/microsoft/PowerToys/blob/37132c91e65f7e7f5d8e0d62bd81b8e8e2eab8bd/src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/PowerLauncherPage.xaml#L294-L313

Reply:
@mykhailopylyp :
> > I'm not sure where you want this to go
> 
> A checkbox is dynamically rendered based on `AditionalOptions`

I have the following under Microsoft.PowerToys.Run.Plugin.System.Main.cs:

        public IEnumerable<PluginAdditionalOption> AdditionalOptions => new List<PluginAdditionalOption>()
        {
            new PluginAdditionalOption()
            {
                Key = ConfirmSystemCommands,
                DisplayLabel = Properties.Resources.confirm_system_commands,
                Value = false,
            },
        };

I have made all other corrections so far as I can tell, but I still have no "Confirm system commands" setting in the powertoyrun settings.  Still not sure where I have gone wrong.  Sorry for being so much trouble, and I sincerely appreciate your help.

Thanks.

Reply:
You're experiencing a bug that was already there. See my comment below.